src/PostgREST/Request/DbRequestBuilder.hs

```diff
@@ -342,6 +350,9 @@ mutateRequest mutation schema tName ApiRequest{..} pkCols readReq = mapLeft ApiR
     rootOrder = maybe [] snd $ find (\(x, _) -> null x) qsOrder
     combinedLogic = foldr addFilterToLogicForest logic qsFiltersRoot
     body = payRaw <$> iPayload -- the body is assumed to be json at this stage(ApiRequest validates)
+    bodyHasPks = case iPayload of
+      Just ProcessedJSON{payKeys} -> all (`elem` payKeys) pkCols
+      _                           -> True
```

Comment:
Was looking for a workaround for this. It looks like the `_ -> True` part is never executed for Deletes (because no `columns` are implemented). But this will be used to add the `422` response on missing/empty bodies for `PATCH`, so I think it could be left as is right now.

Reply:
Solved this in the last commit, no more Codecov warning.